doc/source/whatsnew/v0.19.0.txt

```diff
@@ -623,6 +623,41 @@ Furthermore:
 - Passing duplicated ``percentiles`` will now raise a ``ValueError``.
 - Bug in ``.describe()`` on a DataFrame with a mixed-dtype column index, which would previously raise a ``TypeError`` (:issue:`13288`)
 
+.. _whatsnew_0190.api.perioddtype:
+
+``PeriodIndex`` now has ``period`` dtype
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``PeriodIndex`` now has its own ``period`` dtype. The ``period`` dtype is a
+pandas extension dtype which extends NumPy dtype like ``"category"`` (:issue:`13941`).
```

Comment:
these don't ``extend`` the ``NumPy`` dtype ``category`` (which doesn't exist), rather they are another pandas extension types like ``category``, ``datetime-tz``